src/sentry/models/sentryapp.py

```diff
@@ -173,7 +177,6 @@ def save(self, *args, **kwargs):
     def is_installed_on(self, organization):
         return SentryAppInstallation.objects.filter(
             organization=organization, sentry_app=self,
-        ).exists()
```

Comment:
Was this removal intentional?

Reply:
@markstory arghh no! I merged the linter's suggestion from https://github.com/getsentry/sentry/pull/22644#discussion_r542307972
Good catch!